app/views/content/life-as-a-teacher/change-careers/support-with-changing-careers/_article.html.erb

```diff
@@ -41,7 +41,15 @@
     <p>There are lots of financial support options available for trainee teachers. For example, you may be able to get bursaries and scholarships, student loans, or a training route without fees. The financial support options vary depending on the teacher training route and eligibility criteria.</p>
     <p>Find out more about the different routes and <a href="/life-as-a-teacher/change-careers/how-to-change-careers-to-become-a-teacher">how to change career to become a teacher</a>.</p>
   </section>
-  <%= render CallsToAction::AdviserComponent.new(
-    text: "An adviser with years of teaching experience can help you change careers to become a teacher. Chat by phone, text, or email as little or often as you need.",
-  ) %>
+</div>
+<div class="row">
+  <section class="container">
+    <section class="image-blocks">
+      <%= render CallsToAction::MailinglistBlockComponent.new %>
```

Comment:
I'd suggest switching the order to advisers (most important for this audience), then mailing list, then routes

But also I think events should be included - important for career changers - maybe instead of mailing list